src/modules/person/persistence/person.scope.integration-spec.ts

```diff
@@ -86,6 +89,36 @@ describe('PersonScope', () => {
                 expect(persons).toHaveLength(0);
             });
         });
+        describe('findBy with orginisation id', () => {
```

Comment:
Der Test wäre aussagekräftiger wenn es mehere Personen gäbe die alle Kontexte an unterschiedlichen Orgas haben. Wenn du dann mit dem Scope eine Orga ID mitgibst, sollten nur Personen returned werden die dieser Orga angehören. Aktuell seedest du eine Person und prüfst dann das eine Person returnd wird. Das würde daher auch successfull sein wenn der Filter garnicht funktioniert

Reply:
I included this test only for completion sake it is not part of my ticket I will just remove it